include/rgl/api/core.h

```diff
@@ -713,6 +713,15 @@ RGL_API rgl_status_t rgl_node_raytrace_configure_non_hits(rgl_node_t node, float
  */
 RGL_API rgl_status_t rgl_node_raytrace_configure_mask(rgl_node_t node, const int8_t* rays_mask, int32_t rays_count);
 
+/**
+ * Modifies RaytraceNode to set beam divergence.
+ * Beam divergence is used to calculate the beam width at the distance of hit point.
+ * Setting beam divergence > 0.0f is required to use query for multi-return results.
+ * @param node RaytraceNode to modify.
+ * @param beamDivergence Beam divergence in radians.
+ */
+RGL_API rgl_status_t rgl_node_raytrace_configure_beam_divergence(rgl_node_t node, float beamDivergence);
```

Comment:
```suggestion
RGL_API rgl_status_t rgl_node_raytrace_configure_beam_divergence(rgl_node_t node, float beam_divergence);
```
I assume snake case should be used here. And `rgl_node_raytrace_configure_non_hits` just has the same mistake.

Reply:
Yes, you are correct, thanks for the catch.